tests/fixtures.py

```diff
@@ -72,6 +74,7 @@
     MOCK_REGISTRY_FILEPATH,
     TEST_GAS_LIMIT,
     INSECURE_DEPLOYMENT_SECRET_HASH,
+    NUMBER_OF_ETH_TEST_ACCOUNTS,
```

Comment:
I think you can remove this since you reverted the line where it's used in 800876

Reply:
✔️